server_test.go

```diff
@@ -93,22 +84,82 @@ func TestServerFromConfig(t *testing.T) {
 	assert.Equal(t, expected, string(body))
 }
 
+func TestPassThroughEnabled(t *testing.T) {
+	targetServer := startTargetServer()
+	defer targetServer.Shutdown(context.TODO())
+
+	viewProxyServer := NewServer("http://localhost:9994")
+	viewProxyServer.Port = 9995
+	viewProxyServer.Logger = log.New(ioutil.Discard, "", log.Ldate|log.Ltime)
+	viewProxyServer.PassThrough = true
+
+	go func() {
+		if err := viewProxyServer.ListenAndServe(); err != nil && err != http.ErrServerClosed {
+			panic(err)
+		}
+	}()
+
+	resp, err := http.Get("http://localhost:9995/hello/world")
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	body, err := ioutil.ReadAll(resp.Body)
+
+	assert.Equal(t, 404, resp.StatusCode)
+	assert.Equal(t, "target: 404 not found", string(body))
```

Comment:
Took me a while to distinguish this test from the one below - the `target: ` prefix means that the error is coming from the target server, and is thus proxied

Reply:
Yeah, I found it a bit subtle as well. Happy to change this to make it more clear if you have any suggestions.

Reply:
Rad, I changed it up slightly. thank you!